src/c/luv_c_function_descriptions.ml

```diff
@@ -1197,13 +1197,13 @@ struct
         (ptr Loop.t @->
          ptr t @->
          exit_cb @->
-         ocaml_string @->
-         ptr string @->
+         ptr char @->
+         ptr (ptr char) @->
          int @->
-         ptr string @->
+         ptr (ptr char) @->
          int @->
          bool @->
-         ocaml_string @->
+         ptr char @->
```

Comment:
You may also be able to restore some of this "signature," and remove some of the manual conversion code in `process.ml`.